alien4cloud-core/src/main/java/alien4cloud/topology/DependencyConflictDTO.java

```diff
@@ -0,0 +1,20 @@
+package alien4cloud.topology;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+/**
+ * Represents a dependency conflict for a source CSAR in a topology.
+ */
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class DependencyConflictDTO {
+
+    private String source;
```

Comment:
Add comment
/** Source is the dependency that has expected (rename?) as a dependency (direct or transitive) format archiveName:archiveVersion  */